src/Engine/EngineBase.php

```diff
@@ -39,6 +39,10 @@ abstract class EngineBase
     public const LANG_NAMES = [
         'az-cyrl' => 'Azərbaycan (qədim yazı)',
         'de-frk' => 'Deutsch (Fraktur)',
+        'en-18-handwritten' => 'English Handwritten (18-19th Century)',
+        'en-print-m1' => 'English Typewritten/Print',
+        'en-print-03' => 'English Typewritten/Print',
```

Comment:
I'm not sure how to make them different. These two models do the same thing, but are named differently, have different CERs and cater to different number of languages. @samwilson what do you suggest?

Reply:
I agree that having them the same is confusing. A user will just see two things that are identical, and have no way to choose between them.

I suggest adding the model names, prefixed with 'Transkribus', e.g.

* Transkribus Balinese balmleaf manuscripts
* Transkribus English Handwriting M3
* Transkribus B2022 English Model M4
* …

I also don't think we should include any Transkribus print models, because we don't want people using this for printed works (due to the cost).